plantdescriptionengine/src/main/java/eu/arrowhead/core/plantdescriptionengine/providedservices/pde_monitor/PingTask.java

```diff
@@ -0,0 +1,144 @@
+package eu.arrowhead.core.plantdescriptionengine.providedservices.pde_monitor;
+
+import eu.arrowhead.core.plantdescriptionengine.alarms.AlarmManager;
+import eu.arrowhead.core.plantdescriptionengine.pdtracker.PlantDescriptionTracker;
+import eu.arrowhead.core.plantdescriptionengine.providedservices.pde_mgmt.dto.PdeSystem;
+import eu.arrowhead.core.plantdescriptionengine.providedservices.pde_mgmt.dto.PlantDescriptionEntry;
+import eu.arrowhead.core.plantdescriptionengine.providedservices.pde_mgmt.dto.Port;
+import eu.arrowhead.core.plantdescriptionengine.utils.Metadata;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import se.arkalix.ServiceRecord;
+import se.arkalix.SystemRecord;
+import se.arkalix.net.http.HttpMethod;
+import se.arkalix.net.http.client.HttpClient;
+import se.arkalix.net.http.client.HttpClientRequest;
+import se.arkalix.query.ServiceQuery;
+
+import java.net.InetSocketAddress;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.TimerTask;
+import java.util.stream.Collectors;
+
+public class PingTask extends TimerTask {
+
+    private static final String MONITORABLE_SERVICE_NAME = "monitorable";
```

Comment:
This constants appears in PdeMonitorService class as well. Maybe you can create a public constant for this somewhere and use that in every place.

Reply:
Fixed